app/src/main/java/hm/orz/chaos114/android/slideviewer/SlideViewerApplication.java

```diff
@@ -19,5 +20,10 @@ public void onCreate() {
 
         AnalyticsManager.initializeAnalyticsTracker(this);
         AnalyticsManager.updateUserProperty();
+
+        System.exit(0);
+
+        String text = null;
+        Log.d("log", "text.length=" + text.length());
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/noboru-i/SlideViewer/blob/115209b5cbe5ed6c6a22ddd245c5f8ebe1d36713/app/src/main/java/hm/orz/chaos114/android/slideviewer/SlideViewerApplication.java#L27"></span>Using ‘Log’ instead of ‘Timber’</td>
    </tr>
  </tbody>
</table>